caasp-istio-proxyv2-image/caasp-istio-proxyv2-image.kiwi

```diff
@@ -30,6 +30,11 @@
             <label name="com.suse.reference" value="registry.suse.com/caasp/v5/istio-proxyv2:%%PKG_VERSION%%"/>
           </suse_label_helper:add_prefix>
         </labels>
+        <environment>
+          <env name="ISTIO_META_INTERCEPTION_MODE" value="TPROXY"/>
+          <env name="ISTIO_META_ISTIO_VERSION" value="%%PKG_VERSION%%"/>
+          <env name="ISTIO_META_ISTIO_PROXY_SHA" value="4230ec31b3944d220407d22729736a871bccf797"/>
```

Comment:
Wow, this is a bit frightening, what makes this value to change? I mean it is a SHA to what?

Reply:
Yes, that's the latest SHA from the code. It is only used for debugging purposes and I added it to be in sync with how upstream is doing it:
https://github.com/cilium/proxy/blob/istio-1.5.4/Dockerfile.istio_proxy#L8